### PR TITLE
Replacing ini_get instances with inigetwrapper usages

### PR DESCRIPTION
**Pull request by th3fallen**
resolves https://github.com/owncloud/core/issues/11074

### CODE REVIEW

**Comment by PVince81** [comment]
Is this still valid ? It looks like the associated ticket was closed: https://github.com/owncloud/core/issues/11074

**Comment by LukasReschke** [comment]
This is still valid but requires a rebase.

@th3fallen Please rebase if you have some time.

**Comment by PVince81** [comment]
Rrrrrrebase please

**Comment by PVince81** [comment]
@LukasReschke can you rebase this ?

**Comment by MorrisJobke** [comment]
@PVince81 @LukasReschke @nickvergessen I rebased this and then also added new occurrences of `ini_get` with the IniWrapper methods.

Please review.

**Comment by PVince81** [comment]
@icewind1991

File: lib/private/server.php [hunk 1]
```diff
@@ -521,6 +521,11 @@ public function __construct($webRoot) {
 				$request
 			);
 		});
+
+		$this->registerService('IniWrapper', function ($c) {
```
Comment (icewind1991) [inline lib/private/server.php]: Needs to be added to the public interface?

**Comment by icewind1991** [comment]
Changes look good otherwise

File: lib/private/server.php (hunk 1 above)
Comment (nickvergessen) [inline lib/private/server.php]: No, this change here is a merge conflict and the service would be registered twice atm.

**Comment by MorrisJobke** [comment]
@PVince81 @icewind1991 @nickvergessen So is this okay for merge or not?

**Comment by icewind1991** [comment]
:+1:

**Comment by nickvergessen** [comment]
No, still duplicates the service

File: lib/private/server.php (hunk 1 above)
Comment (MorrisJobke) [inline lib/private/server.php]: Sorry. Missed this totally :(

**Comment by MorrisJobke** [comment]
@nickvergessen I addressed your comment now ;)

**Comment by MorrisJobke** [comment]
@nickvergessen @LukasReschke Can I get a final review for this so it can be merged?

**Comment by LukasReschke** [comment]
:+1:

**Pull request merged**